src/Docker/MlflowProjectResolver.php

```diff
@@ -45,11 +51,13 @@ public function getMlflowProjectIfAvailable(Component $component, array $tokenIn
         try {
             return $this->sandboxesApiClient->getProject();
         } catch (ClientException $e) {
-            if ($e->getCode() === 404) {
-                return null;
-            }
-
-            throw $e;
+            $this->logger->warning(
+                'Failed to fetch MLflow project details. Access to MLflow artifacts will not be enabled!',
+                [
+                    'exception' => $e,
+                ]
+            );
+            return null;
```

Comment:
I like it 👍 .  We shouldn't let this side business fail the job,  if there's an mlflow issue they'll find out about it when they try to use it.